hack/generator/pkg/astmodel/identifier_factory.go

```diff
@@ -73,6 +75,12 @@ func (factory *identifierFactory) CreateIdentifier(name string, visibility Visib
 	}
 
 	result := strings.Join(caseCorrectedWords, "")
+
+	if _, ok := factory.reservedWords[result]; ok {
+		// This is a reserved word, we need to create a different name
+		return result + "Var"
```

Comment:
Can we come up with a better approach than just suffixing with `Var` ?

How about changing `reservedWords` to a `map[string]string` and hand crafting the replacement? 
Then `package` could become `pkg`, for example.

Reply:
Yeah, did that.

It's _really_ close to what the renames dict is already doing but not quite so I've left it it's own thing for now still (as it happens afterwards and is case sensitive).